src/shim/Initialize.ts

```diff
@@ -58,14 +67,14 @@ function invokePasscodeCallback(passcodeCallback: PasscodeCallbackToPromise, doe
  * @param  {boolean} doesUserExist Denotes whether we need to create or update this user
  * @param  {string}  passcode      Users passcode entry
  */
-function setUserPasscode(doesUserExist: boolean, passcode: string) {
+function setUserPasscode(doesUserExist: boolean, passcode: string): Future<Error, SDKInitializationResult> {
     return getJWT(userJWTCallback)
         .flatMap((jwtToken) => {
             const payload = {
-                type: doesUserExist ? "GEN_DEVICE_KEYS" : "CREATE_USER",
```

Comment:
Since we're re-using an existing message string to mean something different now, this will require that we major version the SDK. If we don't, a user on a prior version of the shim will get a later version of the frame and the message they pass will no longer mean what it used to and break.

Reply:
Noted, we'll have to bump major.